extending-valtimo/document/configuring-search-fields.md

```diff
@@ -11,11 +11,12 @@ There are two methods available to create a search field:
 
 ```json
 {
-    "key": "key_to_search_on",
-    "path": "/some/json/path/to/key",
-    "datatype": "date",
-    "fieldtype": "range",
-    "matchtype": "exact"
+  "title": "title",   
+  "key": "key_to_search_on",
+  "path": "/some/json/path/to/key",
```

Comment:
Klaas' PR states the path changed to doc:path.to.key
Maybe you can change this to reflect that as well?